cluster-autoscaler/cloudprovider/gce/gce_manager.go

```diff
@@ -191,7 +205,11 @@ func (m *GceManager) fetchAllNodePools() error {
 			}
 			m.RegisterMig(mig)
 		}
-		// TODO - unregister migs
+	}
+	for _, mig := range m.getMigs() {
```

Comment:
Do we have an idea how dynamically removing MIGs is going to play with clusterstate? I'm not saying we should do anything here - I would be very much against touching clusterstate from cloudprovider. Just something to think about and test.

Reply:
Should be ok. In every CA iteration we are building new maps. However something may go wrong - need to check it manually once it works.